### DISCUSSION TITLE
Support for custom istio injection labels and values ?

### DISCUSSION THREAD

**Discussion by somejfn** [kiali/kiali#3985] [General]
Hi all,

By default,  the Kiali UI add/remove the _istio-injection_ label then set to _enabled/disabled_ values to setup a namespace for sidecar injection.

With Istio's canary control plane support,  this label is deprecated in favor of the **istio.io/rev** label set to the Istio control plane release value (set by the admin).  I.e.  istio.io/rev=1-9-4

I believe the Kiali CR can set a custom label for injection as defined here: https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L679

But what about the value ?   Or is this ^^^ not doing what I think ?

thanks !

**Comment by lucasponce** [comment]
Oh, it makes all of sense, let me create an issue for this.
Perhaps it's more an implementation detail about how to transfer, but yes, configuration should be the place to change this in case the default is not valid.
